packages/nextjs/components/Activities.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+import { ActivitiesItems, TActivityItemProps, TWinnerItemProps, WinnerItem } from "./ActivitiesItem";
+
+export type TActivitiesProps = {
+  rolls: TActivityItemProps[];
+  winners: TWinnerItemProps[];
+};
+
+export const Activities = ({ rolls, winners }: TActivitiesProps) => {
+  const [currentRolls, setCurrentRolls] = useState<TActivityItemProps[]>([]);
+  const [tabIndex, setTabIndex] = useState<number>(0);
+
+  const handleActivityTab = (index: number) => {
+    return setTabIndex(index);
+  };
+
+  return (
+    <div className="card bg-base-300 shadow-xl mx-10 p-1">
+      <div className="flex tabs drop-shadow-md w-auto  pt-3">
+        <a onClick={() => handleActivityTab(0)} className={`tab tab-bordered ${tabIndex == 0 && "tab-active"}`}>
```

Comment:
I think it's better to be two tables without tabs as it was before.

Or even one table, but with different colors for wins and rigged roll wins, but we need to discuss it

Reply:
Based on UI  [Figma UI](https://www.figma.com/file/D0YZXzXhNtaP8j0lcq2JZR/Dice?type=design&node-id=1%3A2&mode=design&t=9nWbwZgT95vcsicq-1)